src/olympia/addons/tests/test_views.py

```diff
@@ -45,6 +45,7 @@
 from olympia.users.templatetags.jinja_helpers import users_list
 from olympia.versions.models import (
     ApplicationsVersions, AppVersion, Version, VersionPreview)
+from olympia.lib.cache import assert_cache_requests
```

Comment:
The `assert_cache_requests` calls are no longer relevant and should be dropped IMHO.

Reply:
Fixed in a5d2ffd46b